clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py

```diff
@@ -25,7 +25,7 @@ class Extension(GeneralExtension):
 
 
     Creates two containers with samples and controls in Clarity:
-        COVID_PREXT_<timestamp> 
+        COVID_PREXT_<timestamp>
```

Comment:
This comment is no longer according to implementation

Reply:
Good call!